modal.js

```diff
@@ -0,0 +1,11 @@
+const openBtn = document.querySelector(".modal-open-btn");
+const closeBtn = document.querySelector(".modal-close-btn");
+var modal = document.querySelector(".modal-overlay");
```

Comment:
any reason for using var after using const above?

Reply:
No, sorry that was by mistake. Thanks for bringing that to notice. Will change it soon to const.

Reply:
Np